### PR TITLE
Credit Interfaces

### PR DESCRIPTION
**Pull request by Horat1us**

### CODE REVIEW

File: src/Credit/TransferTrait.php [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    {
+        return $this->amount;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function getCurrency(): string
+    {
+        return $this->currency ?? 'UAH';
```
Comment (sashabeton) [inline src/Credit/TransferTrait.php]: Missing test for empty currency

**Review by sashabeton — APPROVED** [review]

**Comment by sashabeton** [comment]
GitHub conversation best
